ipn/ipnlocal/local.go

```diff
@@ -2589,13 +2589,17 @@ func (b *LocalBackend) FileTargets() ([]*apitype.FileTarget, error) {
 	if b.state != ipn.Running || nm == nil {
 		return nil, errors.New("not connected")
 	}
-	if !b.capFileSharing {
-		return nil, errors.New("file sharing not enabled by Tailscale admin")
-	}
-	for _, p := range nm.Peers {
-		if p.User != nm.User {
-			continue
+	/*
+		if !b.capFileSharing {
+			return nil, errors.New("file sharing not enabled by Tailscale admin")
 		}
+	*/
+	for _, p := range nm.Peers {
+		/*
+			if p.User != nm.User {
+				continue
+			}
+		*/
```

Comment:
even with this commented out, peerAPI == "", see comment below about exact location, is peerAPI enabled in integration tests?